packages/liveblocks-client/src/client.ts

```diff
@@ -20,6 +20,17 @@ type EnterOptions<
 > = Resolve<
   // Enter options are just room initializers, plus an internal option
   RoomInitializers<TPresence, TStorage> & {
+    /**
+     * A custom callback that will get wrapped around all emitted
+     * notifications. Typically used in the context of a React 17 (or lower)
+     * application to pass in a reference to
+     * a `ReactDOM.unstable_batchedUpdates` or
+     * `ReactNative.unstable_batchedUpdates` here, to announce to the React
+     * renderer that a series of updates can follow that should lead to
```

Comment:
I'm not sure I understand this sentence 🤔 .

Reply:
Ugh, me neither. Let me rewrite it… 😂

Reply:
@ofoucherot I rewrote the comment! Is this better?

Reply:
Yes 🙏